apps/communications/contacts/js/export/contacts_exporter.js

```diff
@@ -156,20 +155,10 @@ window.ContactsExporter = function ContactsExporter(theStrategy) {
   };
 
   //
-  // Based on the strategy configure the progress display to show a
-  // determinative or indeterminate ui depending on the strategy
-  //
-  var _configureProgress = function _configureProgress() {
-    determinativeProgress =
-      strategy.hasDeterminativeProgress !== undefined &&
-      strategy.hasDeterminativeProgress();
-  };
-
-  //
   // Shows the progress dialog based on the
   //
   var _displayProgress = function _displayProgress() {
-    var progressClass = determinativeProgress ? 'progressBar' : 'spinner';
+    var progressClass = true;
```

Comment:
'true' looks reaaaaallly suspicious as the progressClass value. May I ask you why you changed this line? Are you sure that it works as expected?